proto/sources.proto

```diff
@@ -69,6 +69,7 @@ message Bitbucket {
 
   }
   repeated string repositories = 5;
+  repeated string ignoreRepos = 6;
```

Comment:
I always forget, should protos be `camelCase` or `snake_case`?

We kind of have both but I feel like I'm seeing more snake case.